core/src/main/kotlin/org/evomaster/core/output/naming/GraphQLActionTestCaseNamingStrategy.kt

```diff
@@ -24,6 +25,10 @@ open class GraphQLActionTestCaseNamingStrategy(
         return formatName(nameTokens)
     }
 
+    override fun resolveAmbiguity(individualToName: MutableMap<EvaluatedIndividual<*>, String>, duplicatedIndividuals: MutableSet<EvaluatedIndividual<*>>) {
+        // do nothing at the moment. This will be completed with the experimental params disambiguation method
+    }
```

Comment:
when comments like this, have a `TODO` in the comment. `TODO` tags are specially handled by IntelliJ and Git